src/main/java/seedu/trackascholar/logic/commands/AddCommand.java

```diff
@@ -26,15 +26,15 @@ public class AddCommand extends Command {
             + PREFIX_EMAIL + "EMAIL "
             + PREFIX_SCHOLARSHIP + "SCHOLARSHIP "
             + PREFIX_APPLICATION_STATUS + "APPLICATION STATUS "
-            + "[" + PREFIX_TAG + "TAG]...\n"
+            + "[" + PREFIX_MAJOR + "MAJOR]...\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_NAME + "John Doe "
             + PREFIX_PHONE + "98765432 "
             + PREFIX_EMAIL + "johnd@example.com "
             + PREFIX_SCHOLARSHIP + "NUS Global Merit Scholarship "
             + PREFIX_APPLICATION_STATUS + "pending "
-            + PREFIX_TAG + "friends "
-            + PREFIX_TAG + "computerScience ";
+            + PREFIX_MAJOR + "friends "
```

Comment:
Preferably change to sth else instead of friends?

Reply:
Ok thank you it has been resolved.